libcaf_core/caf/fwd.hpp

```diff
@@ -302,6 +300,7 @@ namespace io {
 class hook;
 class broker;
 class middleman;
+template <class...> class typed_broker;
```

Comment:
As a one-off, this can just be clang-formatted as usualy (and push back the `clang-format on` where it used to be)